### PR TITLE
Fixing end and if errors

### PR DESCRIPTION
**Pull request by devinmatte**
Only occuring for on-floor freshmen

### CODE REVIEW

File: packet/templates/active_packets.html [hunk 1]
```diff
@@ -1,7 +1,5 @@
 {% extends "extend/base.html" %}
 
-{% set can_sign = (info.onfloor and info.uid != packet.rit_username) or info.realm == "csh" %}
```
Comment (stevenmirabito) [inline packet/templates/active_packets.html]: Should replace this with `info.onfloor or info.realm == "csh"` as used below, then restore the `can_sign` usage on line 23 and remove the new definition on line 28.

File: packet/templates/active_packets.html [hunk 2]
```diff
@@ -20,13 +18,14 @@ <h3 class="page-title">Active Packets</h3>
                                     <tr>
                                         <th>Name</th>
                                         <th>Signatures</th>
-                                        {% if can_sign %}
```
Comment (stevenmirabito) [inline packet/templates/active_packets.html]: See above.

File: packet/templates/active_packets.html [hunk 3]
```diff
@@ -20,13 +18,14 @@ <h3 class="page-title">Active Packets</h3>
...
+                                        {% if info.onfloor or info.realm == "csh" %}
                                             <th>Sign</th>
                                         {% endif %}
                                     </tr>
                                     </thead>
                                     <tbody>
                                     {% for packet in packets %}
+                                        {% set can_sign = info.onfloor or info.realm == "csh" %}
```
Comment (stevenmirabito) [inline packet/templates/active_packets.html]: See above.

File: packet/member.py [hunk 4]
```diff
@@ -117,7 +117,7 @@ def query_signed_intromember(member):
         return db.engine.execute("""
             SELECT DISTINCT packet.freshman_username AS username, signature_fresh.signed AS signed FROM packet 
             INNER JOIN signature_fresh ON packet.id = signature_fresh.packet_id 
-            WHERE signature_fresh.freshman_username = """ + member + ";")
+            WHERE signature_fresh.freshman_username = '""" + member + "';")
```
Comment (stevenmirabito) [inline packet/member.py]: Hello, SQL injection, my old friend. Is `member` something internal or is it coming from an external source, e.g. the URL?

Comment (devinmatte) [reply packet/member.py]: Looks like member in this case is only ever fetched from OIDC UID

**Pull request merged**